test/test_suit/test_snowpipe_streaming_string_json.py

```diff
@@ -31,10 +31,19 @@ def send(self):
             print("Sending in Partition:" + str(p))
             key = []
             value = []
-            for e in range(self.recordNum):
+            for e in range(self.recordNum - 1):
```

Comment:
why did we change this?

Reply:
we are sending the tombstone record as the last record. The schema evolution tests rely on exactly recordNum rows being sent (throws [NonRetryableError](https://github.com/snowflakedb/snowflake-kafka-connector/blob/master/test/test_suit/test_schema_evolution_json.py#L84)). Additionally a majority of the tests rely on row count as a buffer threshold, so sending the tombstone record as an additional row would increase test runtime

Reply:
I see, thanks. 
why do we have to make changes in normal snowpipe streaming tests? we can do this change in tombstone specific tests right?

Reply:
yes i could add a tombstone specific test, but i didn't want to extend the e2e runtime. plus tombstone ingestion is enabled by default, so i think it makes sense to put it in the normal e2e tests

happy to copy this into a new test if you prefer

Reply:
Oh I see, thanks for explanation, i missed it is default enabled. please add a comment wherever possible. Thanks

Reply:
Good point it is confusing, added comments to the tests